src/Scope.ts

```diff
@@ -762,6 +790,13 @@ export class Scope {
             const ifaceStmt = pair[1]?.item;
             ifaceStmt?.buildSymbolTable(this.getParentInterface(ifaceStmt));
         }
+
+        //also link enums
+        const enumMap = this.getEnumMap();
```

Comment:
I don't think it strictly necessary to build the symbol tables here instead of at parse time, but i don't think it hurts

Reply:
I like it being here because that ensures we support future enum value types that come from other files.